tools/modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -154,4 +156,22 @@ sub _vep_config_warning {
   warn $message." - thrown by ".__FILE__."\n";
 }
 
+sub _configure_assembly_converter_files {
+  my $self = shift;
+  my $species = $self->species;
+  my $version = $SiteDefs::ENSEMBL_VERSION;
+  my $chain_file_dir = File::Spec->catfile($SiteDefs::ENSEMBL_CHAIN_FILE_DIR, $species);
+  if (-d $chain_file_dir) {
```

Comment:
does that mean if a species doesnt have assembly converter files, it wont have a directory. there wont be a case where it will be an empty directory (specially if there was one in previous release and in the new release only the files have been deleted leaving an empty directory? )

Reply:
I believe there will be no empty dirs. I can confirm this with Production.

Reply:
Confirmed with production, no empty dirs. https://genomes-ebi.slack.com/archives/C0F4FQPHN/p1659604865781979?thread_ts=1659603653.602389&cid=C0F4FQPHN